lib/charms/observability_libs/v1/kubernetes_compute_resources_patch.py

```diff
@@ -0,0 +1,631 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""# KubernetesComputeResourcesPatch Library.
+
+This library is designed to enable developers to more simply patch the Kubernetes compute resource
+limits and requests created by Juju during the deployment of a sidecar charm.
```

Comment:
the reference to 'sidecar charm' is kind of old-fashioned. It's just a charm these days.